apps/designer/app/designer/shared/fonts-manager/item-menu.tsx

```diff
@@ -42,6 +44,13 @@ const ItemMenu = ({
     };
   }, []);
 
+  const [authPermit] = useAuthPermit();
+
+  const deleteDisabled = authPermit === "view";
```

Comment:
I try to use is/has/can for all booleans for clarity

```suggestion
  const isDeleteDisable = authPermit === "view";
```